plugins/modules/delete_service_account_by_id.py

```diff
@@ -16,7 +16,7 @@
 
 short_description: Delete a Service Account for use with Red Hat Openshift Application Services.
```

Comment:
```suggestion
short_description: Delete a Service Account for use with Red Hat OpenShift Application Services.
```
We could do a quick search for other places where the same capitalization can be done.